superset_patchup/__init__.py

```diff
@@ -1,5 +1,14 @@
 """
 Main init file for superset_patchup
 """
-VERSION = (0, 1, 4)
-__version__ = ".".join(str(v) for v in VERSION)
+from .version import (VERSION, __version__) # noqa
+
+
+# pylint: disable=unused-argument
+def patch_app(superset_app):
+    """Hook called once superset is initialized"""
+    # DRAGONS - import here, otherwise we add dependencies for packaging which
```

Comment:
This is also super confusing - our setup script imports superset_patchup, but needs the standard import to be very minimal since it's only pulling the version.